tools/metactl/src/grpc.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_meta_grpc::MetaGrpcClient;
+use common_meta_types::protobuf::Empty;
+use tokio_stream::StreamExt;
+
+pub async fn export_meta(addr: &str) -> anyhow::Result<()> {
+    let client = MetaGrpcClient::try_create(addr, "root", "xxx", None, None).await?;
```

Comment:
password need parameter setting?

Reply:
I find this use case in other codes. Wondering it needs an empty user/password or this(root/xxx) if connected from localhost.